tree-diff

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
I am not sure about this solution. The problem I see here is that it will result in different Python for everyone. What about being specific like `python3.7`? Maybe we could also agree on the Python interpreter we want to use, because Fedora 31 already comes with 3.8 and I am definitely going to upgrade ASAP.

Reply:
> The problem I see here is that it will result in different Python for everyone.

This can happen both in case of `/usr/bin/python3` and `/usr/bin/env python3`, right? I'm not sure if I want to hardcore the `python3.7` binary. That would mean this script wouldn't be able to run on machines with python 3.8+... I don't know what is the best practice for handling different interpreter versions in Python community.

Reply:
Yeah, there is no perfect solution :-). A recommended one (from our Python maintenance team) is to use setup tools to generate the hashbang during build, but that is not exactly applicable in this case, because it is a standalone script.